backend/tracim_backend/tests/functional/test_webdav.py

```diff
@@ -84,44 +84,25 @@ class TestFunctionalWebdavGet(object):
     Test for all Webdav "GET" action in different case
     """
 
-    def test_functional__webdav_access_to_root__nominal_case(
-        self, session, user_api_factory, webdav_testapp
+    @pytest.mark.parametrize(
+        "login", ("test@test.test", "TEST@TEST.TEST", "TeSt@tEsT.teST", "testuser")
+    )
+    def test_functional__webdav_access_to_root__nominal_cases(
+        self, session, user_api_factory, webdav_testapp, login
     ) -> None:
 
         uapi = user_api_factory.get()
 
         uapi.create_user(
-            "test@test.test",
+            email="test@test.test",
             password="test@test.test",
+            username="testuser",
             do_save=True,
             do_notify=False,
             profile=Profile.USER,
         )
         transaction.commit()
-        webdav_testapp.authorization = ("Basic", ("test@test.test", "test@test.test"))
-        # check availability of root using webdav
-        res = webdav_testapp.get("/", status=200)
-        assert res
-
-    def test_functional__webdav_access_to_root__insensitive_case_email(
```

Comment:
No sure to understand the diff, you have merged 2 test here ?

Reply:
Yes, by using parameters for it.

Reply:
> i do suggest rename this variable to login as login is the name used for this kind of thing.

Please see my answer to the same remark by @raphj: https://github.com/tracim/tracim/pull/3077#discussion_r427170323

Reply:
ok